README.md

```diff
@@ -280,6 +287,17 @@ Install kubeflow roles:
 kustomize build common/kubeflow-roles/base | kubectl apply -f -
 ```
 
+#### Kubeflow Istio Resources
+
+Create the Kubeflow Gateway, `kubeflow-gateway` and ClusterRole, 
+`kubeflow-istio-admin`.
+
+Install kubeflow istio resources:
+
+```sh
+kustomize build common/istio-1-22/kubeflow-istio-resources/base | kubectl apply -f -
```

Comment:
Please cross check this with example/kustomization.yaml

Reply:
Looks good in https://github.com/kubeflow/manifests/blob/da0255f10d875040c2d845cd61b7938236c0dfaa/example/kustomization.yaml#L38

Reply:
Looks also good https://github.com/kubeflow/manifests/blob/da0255f10d875040c2d845cd61b7938236c0dfaa/example/kustomization.yaml#L59